packages/taquito/src/contract/rpc-contract-provider.ts

```diff
@@ -395,6 +400,127 @@ export class RpcContractProvider extends Provider implements ContractProvider, S
     return new TransactionOperation(hash, content, source, forgedBytes, opResponse, context);
   }
 
+  /**
+   *
+   * @description Stake tz from current address to a specific address. Built on top of the existing transaction operation
```

Comment:
can consider the octez-client desc `Stake the given amount for the source` and all places with @description